provider/sriov/sriov_base.py

```
@@ -192,6 +192,9 @@ def parse_iface_dict(self):
             if vf_pci_addr2.get('type'):
                 del vf_pci_addr2['type']
             iface_dict = eval(self.params.get('hostdev_dict', '{}'))
+        driver_dict = eval(self.params.get("driver_dict", "{}"))
```

Ruff (0.15.2)

[error] Use of possibly insecure function; consider using `ast.literal_eval`

(S307)